CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Main
 
+* [6317](https://github.com/grafana/loki/pull/6317/files) **dannykoping**: General: add cache usage statistics
```

Comment:
nitpicking: although this is a new thing for the users, I feel like it's not CHANGELOG-worthy, so WDYT of not having this in the changelog? In case you prefer to keep it, IDK why all these entries are listed under `Main` instead of unreleased but I think this one should be under `Unreleased/All Changes/Loki/Enhancements`